website/content/docs/operations/garbage-collection.mdx

```diff
@@ -0,0 +1,212 @@
+in a terminal state for at least four hours before the garbage collection
+process permanently deletes the job and its dependent objects.
+
+Manually triggering garbage collection by running the `nomad system gc` command
+instructs the server leader to ignore thresholds and immediately purge all
+terminal objects.
+
+## Client-side garbage collection
+
+On each client node, Nomad must clean up resources from terminated allocations
+to free disk and memory on the machine.
+
+### Configuration
+
+These settings govern allocation garbage collection behavior on each client node.
+
+| Parameter | Default | Description  |
+| -------- | ------- | ------------- |
+| [`gc_interval`](/nomad/docs/configuration/client#gc_interval)  | 1m  | Interval at which Nomad attempts to garbage collect terminal allocation directories |
+| [ `gc_disk_usage_threshold` ](/nomad/docs/configuration/client#gc_disk_usage_threshold)  | 80 | Disk usage percent which Nomad tries to maintain by garbage collecting terminal allocations |
+| [ `gc_inode_usage_threshold` ](/nomad/docs/configuration/client#gc_inode_usage_threshold) | 70 | Inode usage percent which Nomad tries to maintain by garbage collecting terminal allocations |
+| [ `gc_max_allocs` ](/nomad/docs/configuration/client#gc_max_allocs) | 50 | Maximum number of allocations which a client will track before triggering a garbage collection of terminal allocations |
+| [ `gc_parallel_destroys ` ](/nomad/docs/configuration/client#gc_parallel_destroys) | 2 | Maximum number of parallel destroys allowed by the garbage collector |
+
+Refer to the [client block in agent configuration
+reference](/nomad/docs/configuration/client) for complete parameter descriptions
+and examples.
+
+Note that there is no time-based retention setting for allocations. Unlike jobs
+or evaluations, you cannot specify a time to keep allocations alive before
+garbage collection. As soon as an allocation is terminal, it becomes eligible
+for cleanup if the configured thresholds demand it.
+
+### Triggers
+
+Nomad's client runs allocation garbage collection based on these triggers:
+
+- Periodic timer
+
+  The garbage collector launches a ticker based on the configured
+  `gc_interval`. On each tick, the garbage collector checks to see if it needs
+  to remove allocations.
+
+- Terminal state
+
+  When an allocation transitions to a terminal state, Nomad marks
+  the allocation for garbage collection and then signals the garbage collection
+  process to run immediately.
+
+- Allocation placement
+
+  Nomad may preemptively run garbage collection to make room for new
+  allocations. The client garbage collects older,
+  terminal allocations if adding new allocations would exceed the
+  `gc_max_allocs` limit.
+
+- User-initiated server garbage collection
+
+  When you initiate garbage collection on the server by running the `nomad
+  system gc` command, the clients collect all eligible allocations, bypassing
```

Comment:
This gets triggered by any server-side garbage collection, not just the user-initiated version of it.